src/IrisConvert/IrisConvert.php

```diff
@@ -103,23 +121,21 @@ public function createArrayOfSizesToBeConverted($metadata)
     public function convertArrayOfSizes()
     {
         // Remove wp_generate_attachment_metadata action
-        remove_filter('wp_generate_attachment_metadata', [Iris::class, 'convertImageOnUplodad'], 10);
+        // remove_filter('wp_generate_attachment_metadata', [Iris::class, 'convertImageOnUplodad'], 10);
         switch ($this->file_ext) {
-
             case 'jpeg':
             case 'jpg':
-
                 foreach ($this->array_of_sizes_to_be_converted as $key => $value) {
                     $image = imagecreatefromjpeg($value);
                     if (0 === $key) {
                         $webp_file = $this->file_dirname . '/' . $this->file_name_no_ext . '.webp';
                         imagewebp($image, $webp_file, 80);
+                        // self::setOriginalImage($this->image_metadata, $this->file_name_no_ext . '.webp');
```

Comment:
Should this be commented out?

Reply:
Yes and No, I think I'll remove that method.